src/network/networkd-network.c

```diff
@@ -592,7 +592,7 @@ int network_load(Manager *manager, OrderedHashmap **networks) {
 
         STRV_FOREACH(f, files) {
                 r = network_load_one(manager, networks, *f);
-                if (r < 0)
+                if (r < 0 && !ERRNO_IS_PRIVILEGE(r))
                         return log_error_errno(r, "Failed to load %s: %m", *f);
```

Comment:
hmm, so something's afoot regarding logging here. According to our usual coding style functions should either log about all errors or about none (in which case the caller should do that). 

Previously we'd log here about all errors, thus implying that network_load_one() would not log about any (which is not correct, is it?).

With the new code we'll now log about some errors, but not about others, which seems to contradict the rule quite a bit.

i.e. it appears to me there are two issues here: 

1. we should clean up logging so that every error is logged once and once only
2. we should eat up ERRNO_IS_PRIVILEGE() errors after LOG_WARNING about them, so that we continue running if we encounter an issue like that.

Reply:
actually, I wonder if it wouldn't be more appropriate to continue running regardless of any error with loading configuration files. i.e. if some unexpected error happens while loading some .netdev it should not take down networkd as a whole and other netdevs...

i.e. the change should cover more than ERRNO_IS_PRIVILEGE, no?